src/Data/Formatter/DateTime.purs

```diff
@@ -201,10 +209,14 @@ formatF cb dt@(DT.DateTime d t) = case _ of
     (if (fromEnum $ T.hour t) >= 12 then "PM" else "AM") <> cb a
   Minutes a →
     show (fromEnum $ T.minute t) <> cb a
+  MinutesTwoDigits a →
+    (padSingleDigit <<< fromEnum $ T.minute t) <> cb a
   Seconds a →
     show (fromEnum $ T.second t) <> cb a
+  SecondsTwoDigits a →
+    (padSingleDigit <<< fromEnum $ T.second t) <> cb a
   Milliseconds a →
-    show (fromEnum $ T.millisecond t) <> cb a
```

Comment:
@garyb Yep, it would be cool to pad milliseconds up to 3 digits, similar as `momentjs` it does by using formats of `S` `SS` `SSS` (https://momentjs.com/docs/#hour-minute-second-millisecond-and-offset-tokens) 

However, the goal of this pull request was just fixing the format of minutes / seconds, but not adding this extra feature. Please let do this with another pull request, I'm willing to do this if there a need. Thx!